### DISCUSSION TITLE
Add tutorial models to muse `example` folder?

### DISCUSSION THREAD

**Discussion by tsmbland** [EnergySystemsModellingLab/MUSE_OS#314] [General]
Just a thought, and maybe this isn't a good idea, but wondering if it might be worth moving the model files for the tutorials over to the `example` folder in `src/muse/data`.

I have a couple of reasons for this

Firstly, since many of the tutorials build on each other, any errors/differences that the user makes in one tutorial will propogate to the next one, and they could quite easily end up in a confusing mess if they don't follow the instructions to the letter. It would be nice if users could at least have the correct starting point for each tutorial by running for example, `muse --model tutorial_add_solar --copy`, which they would be able to do if this model was added to the `example` folder. (The alternative would be to change each tutorial to build directly on one of the existing example models rather than building on a previous tutorial, which I'm also considering). In any case, it would nice to give the user easy access to a source of 'ground truth' without having to clone the whole repo or poke around github.

Another benefit is that it would simplify the testing suite by having all the models for the regression tests in one place. (But this is a bit irrelevant since the tests are already set up successfully as they are.)

We'd also need to think about where to keep the results files. The results for the current example models are currently kept in `tests/example_outputs`, so we could just add the tutorial results here too, but then I'd probably suggest moving this folder to the top level of the repo so it's a bit easier to access from the notebooks. Or would they need to be in the `docs` folder for the documentation build to work?

Not really sure what the downsides of this would be (apart from increasing the size of the `src` folder slightly), but maybe I'm missing something? @dalonsoa @alexdewar

**Comment by tsmbland** [comment]
Actually, on second thought, we wouldn't even need to store the model files as they can be generated on the fly using the scripts I've written, so we'd just need to move the scripts to somewhere in the `src` folder and modify the `copy_model` function in `examples.py` to point the the appropriate functions... 

Might give it a go as a draft PR and see what it would look like.

**Reply by alexdewar** [reply]
This sounds sensible to me, but I guess the devil is always in the detail with these things! Looking forward to seeing the PR

**Comment by tsmbland** [comment]
I'm not going to bother with this since the tutorials no longer build on each other so there's no real benefit
